dotCMS/src/main/webapp/html/portlet/ext/workflows/schemes/workflow_main.jsp

```diff
@@ -20,26 +20,37 @@
-				.content h2{font-size:200%;}
-				.content p{margin:0;}
-				.content ul{margin:5px 0 25px 15px;padding:0 0 0 10px;list-style-position:outside; list-style:decimal;}
-				.content li{list-style-position:outside; list-style:disc;}
-				.content a{color:#fff;}
-			</style>
-			<div class="greyBg"></div>
-			<div class="wrapper">
-				<div class="content">
-					<h2><%= LanguageUtil.get(pageContext, "Workflows") %></h2>
-					<p><%= LanguageUtil.get(pageContext, "Workflows-Not-Licensed") %></p>
-				</div>
-			</div>
-	    </div>
+        <style>
+            #dotAjaxMainHangerDiv, #dotAjaxMainDiv { height: 100%; }
+            .portlet-wrapper { display: flex; height: 100%; justify-content: center; }
+            .unlicense-content { align-self: center; text-align: center; }
+            .unlicense-content i, .unlicense-content h4 { color: #b3b1b8 }
+            .unlicense-content h4 { font-size:24px; font-weight:400; line-height:36px; }
+            .unlicense-content p { font-size:14px; font-weight:400; line-height:21px; margin-block-end:14px; margin-block-start:14px; }
+            .unlicense-content ul { display: inline-block; margin-block-end: 14px; margin-block-start: 14px;}
+            .unlicense-content ul li { line-height: 21px; list-style-type:disc; text-align: left; }
+            .unlicense-content ul li a { color: var(--color-main); font-size:14px; }
+            .request-license-button { background: var(--color-main); border: solid 1px transparent; border-radius: 2px; color: #fff; display: inline-block; line-height: 36px; padding: 0 24px; text-decoration: none; text-transform: uppercase; }
+            .request-license-button:hover { background: var(--color-main_mod); border: solid 1px transparent; }
+        </style>
+        <div class="portlet-wrapper">
+            <div class="unlicense-content">
+                <i class="material-icons" style="font-size: 120px;">device_hub</i>
+                <h4><%=LanguageUtil.get(pageContext, "com.dotcms.repackage.javax.portlet.title.workflow-schemes")%></h4>
+                <p><%=LanguageUtil.get(pageContext, "com.dotcms.repackage.javax.portlet.title.workflow-schemes")%> <%=LanguageUtil.get(pageContext, "only-available-in-enterprise")%></p>
+                <ul>
+                    <li>
+                        <a target="_blank" href="https://dotcms.com/product/features/feature-list"><%=LanguageUtil.get(pageContext, "Learn-more-about-dotCMS-Enterprise")%></a>
+                    </li>
+                    <li>
+                        <a target="_blank" href="https://dotcms.com/contact-us/"><%=LanguageUtil.get(pageContext, "Contact-Us-for-more-Information")%></a>
+                    </li>
+                </ul>
+                <p style="display: block;">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5587482)